.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Run CI
+on:
+  push:
+    branches: [ v8 ]
+    paths-ignore:
+      - '**.md' # Do not need to run CI for markdown changes.
+  pull_request:
+    branches: [ v8 ]
+    paths-ignore:
+      - '**.md'
```

Comment:
Wondering this for other SDKs as well. Do you think it should run daily/weekly so that incompatibilities don't slip in (like the Github runner tools subtly changing?)

Reply:
Yes, I was actually thinking about this as well. This is my last PR for release please in my repos, so I'm going to work on getting this one merged, and then probably come back through everyone one final time.